src/commons/sagas/GitHubPersistenceSaga.ts

```diff
@@ -58,11 +58,6 @@ function* githubLoginSaga() {
 }
 
 function* githubLogoutSaga() {
-  if (store.getState() && store.getState().workspaces.githubAssessment.hasUnsavedChanges) {
-    yield call(showWarningMessage, 'You have unsaved changes!', 2000);
-    return;
-  }
-
   yield put(actions.removeGitHubOctokitObjectAndAccessToken());
   yield call(showSuccessMessage, `Logged out from GitHub`, 1000);
 }
```

Comment:
Ditto, I'll keep these sagas just in case (leaving them in won't break anything), but we can remove in the future if the folders team do not require them.